examples/building.rs

```diff
@@ -0,0 +1,233 @@
+use std::net::SocketAddr;
+use std::sync::atomic::{AtomicUsize, Ordering};
+
+use log::LevelFilter;
+use num::Integer;
+use valence::biome::Biome;
+use valence::block::BlockState;
+use valence::chunk::{Chunk, UnloadedChunk};
+use valence::client::event::DiggingStatus;
```

Comment:
```suggestion
use valence::client::DiggingStatus;
```

Reply:
`valence::protocol::packets::c2s::play::DiggingStatus` conflicts with `valence::client::DiggingStatus`, so making this change makes it not compile. I'm going to go ahead and remove that import and just use the fully qualified name `valence::protocol::packets::c2s::play::DiggingStatus` in `client.rs`